pyproject.toml

```diff
@@ -1,11 +1,11 @@
 [tool.poetry]
-name = "ttyg-evaluation"
-version = "2.1.1"
-description = "Talk to Your Graph (TTYG) Evaluation is a Python module for evaluating whether LLM agents correctly orchestrate and invoke available tools to answer user questions, based on a gold-standard corpus of tool call expectations."
+name = "qa-eval"
+version = "2.1.2"
```

Comment:
Both #9 and #10 introduce breaking changes, if we want to release them together we should release a new major version, i.e. `3.0.0`

Reply:
Bumped version to `3.0.0`